cypress/integration/e2e/article.metaAndOphan.e2e.spec.js

```diff
@@ -0,0 +1,118 @@
+import { getPolyfill } from '../../lib/polyfill';
+import { disableCMP } from '../../lib/disableCMP.js';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+
+describe('E2E Page rendering', function () {
+	before(getPolyfill);
+
+	beforeEach(function () {
+		disableCMP();
+		setLocalBaseUrl();
+	});
+
+	const getExists = (selector) => cy.get(selector).should('have.length', 1);
+
+	describe('for WEB the document renders with the correct meta and analytics info', function () {
+		it(`Has a head element`, function () {
+			cy.visit(
+				`/Article?url=https://www.theguardian.com/lifeandstyle/2021/jan/21/never-conduct-any-business-naked-how-to-work-from-bed-without-getting-sacked`,
+			);
+			// cy.get(`head`).should('have.length', 1);
+			getExists(`head`);
```

Comment:
I prefer to have as much code inline for tests so that they're easier to scan when debugging. I find abstractions in tests especially costly but this is a subjective thing so 🤷

Reply:
Yeah, I tend to agree but I think in this case as there's so many it actually makes the test much easier to read as a whole